client/signup/steps/theme-selection/index.jsx

```diff
@@ -58,8 +59,7 @@ class ThemeSelectionStep extends Component {
 			<SignupThemesList
 				surveyQuestion={ this.props.chosenSurveyVertical }
 				designType={ this.props.designType || this.props.signupDependencies.designType }
-				handleScreenshotClick={ this.pickTheme }
-			/>
```

Comment:
i'd generally go for a new line for symmetry but eslint doesn't express an opinion on the matter.